examples/attach/BUILD.bazel

```diff
@@ -10,7 +17,7 @@ oci_image(
 
 cosign_attach(
     name = "attach",
-    attachment = "sbom.spdx",
+    attachment = ":sbom_generated.spdx",
     image = ":image",
     repository = "test",
     type = "sbom",
```

Comment:
don't you need to use the external repo in one of these attributes to see a potential failure?

Reply:
I added the test for signing. I'll add a test for this one as well.

Reply:
done